docs/_documentations/mdt-eclipse-codechange.md

```diff
@@ -1,36 +1,35 @@
 ---
 layout: docs
-title: "Getting started with Codewind for Eclipse"
-description: "Getting started with Codewind for Eclipse"
+title: "Making a code change with Eclipse"
+description: "Making a code change with Eclipse"
 keywords: introducing, introduction, overview, tools, get, getting, start, started, install, vscode, visual, studio, code, Codewind for VS Code getting started, VS Code Marketplace, VS Code Extensions view, VS Code workspace,installing Codewind for VS Code
 duration: 1 minute
 permalink: mdt-eclipse-codechange
 type: document
 order: 1
 parent: mdt-vsc-overview
 ---
-# Making a code change
+# Making a code change with Eclipse 
 <br/>
-Codewind will automatically build and redeploy your application whenever you make a code change and save the file.
+Codewind automatically builds and redeploys your application whenever you make a code change and save the file.
 
-To see this in action, we can make a change to the getting started example.
+To see this feature in action, make a change to the getting started example.
 
-First step is to edit the index.html file
+1\. Edit the index.html file.
 
 ![](dist/images/eclipsechangeproject1.png){:width="800px"}
 
-and navigate to the bottom of the file to find the lines
+2\. Navigate to the bottom of the file to find the lines.
```

Comment:
Here's what Acrolinx says about the word "Navigate": `Use only when a more direct verb, such as "click," "select," "browse," or "go to," is not applicable. Use only to provide direction to a location (for example, "Navigate to the directory that contains the files that you want to edit").` In this case, "go" works great. :)